packages/next/src/build/webpack/loaders/next-image-loader/blur.ts

```diff
@@ -37,7 +37,7 @@ export async function getBlurImage(
   let blurWidth: number = 0
   let blurHeight: number = 0
 
-  if (VALID_BLUR_EXT.includes(extension)) {
+  if (VALID_BLUR_EXT.includes(extension) && !isAnimated(content)) {
```

Comment:
Since this will cause `blurDataURL` to be undefined, I think we also need a better error message during `next dev`.

https://github.com/vercel/next.js/blob/df6ec96ab2937b68e7c4a6830d11f12aa4c609e6/packages/next/src/shared/lib/get-img-props.ts#L487-L500

Reply:
Would you feel it sufficient to add `(animated images not supported)` to the end of the `Change the "src" property to a static import with one of the supported file types` lines?

Reply:
Yeah on the second bullet point

Reply:
Also the docs need to be updated here https://github.com/vercel/next.js/blob/df6ec96ab2937b68e7c4a6830d11f12aa4c609e6/errors/placeholder-blur-data-url.mdx#L11